santa/src/santa.cpp

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) 2018 Trail of Bits, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <sstream>
+#include <string>
+
+#include <boost/iostreams/filter/gzip.hpp>
+#include <boost/iostreams/filtering_streambuf.hpp>
+
+#include <sqlite3.h>
+
+#include "santa.h"
+
+const std::string LOG_PATH = "/var/db/santa/santa.log";
```

Comment:
Are the Santa log and rules always at a fixed location or are they configurable? (just want to check that they are constant)

Reply:
fixed location: https://github.com/google/santa/blob/d1d9762e29731c1857de25b40ebfe3b2b4b19bb0/Source/santad/SNTDatabaseController.m#L26